modules/integration/tests-integration/tests-backend/src/test/resources/org/wso2/identity/integration/test/scim2/rest/api/claim-management-update-emailverified-claim.json

```diff
@@ -0,0 +1,17 @@
+{
+  "claimURI": "http://wso2.org/claims/identity/emailVerified",
+  "description": "Email Verified",
+  "displayOrder": 0,
+  "displayName": "Email Verified",
+  "readOnly": false,
+  "regEx": "",
+  "required": false,
+  "supportedByDefault": %b,
```

Comment:
Is the %b intentional?

Reply:
Yes, this value is replaced at the test execution time in the test class.